actionpack/test/dispatch/cookies_test.rb

```diff
@@ -1235,6 +1240,33 @@ def test_cookies_are_not_cleared
     assert_equal "bar", @controller.encrypted_cookie
   end
 
+  def test_signed_cookie_with_expires_set_relatively
+    cookies.signed[:user_name] = { value: "assain", expires: 2.hours }
+
+    travel 1.hour
+    assert_equal "assain", cookies.signed[:user_name]
+
+    travel 2.hours
+    assert_nil cookies.signed[:user_name]
+  end
+
+  def test_encrypted_cookie_with_expires_set_relatively
+    cookies.encrypted[:user_name] = { value: "assain", expires: 2.hours }
+
+    travel 1.hour
+    assert_equal "assain", cookies.encrypted[:user_name]
+
+    travel 2.hours
+    assert_nil cookies.encrypted[:user_name]
+  end
+
+  def test_vanilla_cookie_with_expires_set_relatively
+    travel_to Time.utc(2017, 8, 15) do
+      get :cookie_expires_in_two_hours
+      assert_cookie_header "user_name=assain; path=/; expires=Tue, 15 Aug 2017 02:00:00 -0000"
```

Comment:
I used `Time.utc` method, since there were other tests using it while setting the `:expires` option. Was that method the root of your concern? :sweat_smile: